work2/qqCatOwlbb/MyAnimalShop.java

```diff
@@ -0,0 +1,106 @@
+package com.catowl.animalshop;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.time.LocalDate;
+import java.util.zip.CheckedOutputStream;
+
+public class MyAnimalShop implements AnimalShop {
+    private double balance;
+    ArrayList<Animal> animalsList;
+    ArrayList<Customer> customersList;
+    boolean status;
```

Comment:
```suggestion
    ArrayList<Animal> animalsList;
    ArrayList<Customer> customersList;
    boolean status;
```
这里的访问权限怎么不写，虽然默认是private，但是也要写一下的